include/dfrn.php

```diff
@@ -865,6 +865,30 @@ private static function entry($doc, $type, $item, $owner, $comment = false, $cid
 	}
 
 	/**
+	 * @brief encrypts data via AES
+	 *
+	 * @param string $data The data that is to be encrypted
+	 * @param string $key The AES key
+	 *
+	 * @return string encrypted data
+	 */
+	private static function aes_encrypt($data, $key) {
```

Comment:
Since they are used by two different connectors, it seems a very good idea to me to pull it out of either in the shared `crypto.php` library.